.github/workflows/ci_tests.yml

```diff
@@ -114,15 +121,20 @@ jobs:
       - name: Setup Macos
         if: startsWith(matrix.platform.os, 'macos')
         run: sudo chmod -R 777 /opt/
-      - name: Print installed softwares
+      - name: Print installed software
         shell: bash
         run: |
           echo "Build system:"
           cmake --version
           ninja --version
       - name: Configure CMake
-        run: |
-          cmake -B build -S . -DCMAKE_CXX_STANDARD=${{ matrix.cpp_version }} -DCMAKE_TOOLCHAIN_FILE="${{ matrix.platform.toolchain }}" ${{ matrix.cmake_args.args }}
+        run: 'cmake
+                 -B build
+                 -S .
+                 -DCMAKE_CXX_STANDARD=${{ matrix.cpp_version }}
+                 -DCMAKE_TOOLCHAIN_FILE="${{ matrix.platform.toolchain }}"
+                 -DCMAKE_PROJECT_TOP_LEVEL_INCLUDES="./cmake/use-fetch-content.cmake"
```

Comment:
Let me note since the formatting changes obscure the change a little. This argument referencing `use-fetch-content.cmake` is added. It makes this CI workflow use `FetchContent`.

Pay attention to the other `cmake` commands in this file as well. If you do not see a mention of `use-fetch-content.cmake`, the workflow uses `find_package` and system installed dependencies.